examples/express/server.js

```diff
@@ -0,0 +1,143 @@
+  Connection, 
+  Keypair, 
+  PublicKey, 
+  SystemProgram, 
+  Transaction, 
+  clusterApiUrl, 
+  LAMPORTS_PER_SOL 
+} from "@solana/web3.js";
+
+const DEFAULT_SOL_ADDRESS = Keypair.generate().publicKey;
+const DEFAULT_SOL_AMOUNT = 1;
+const connection = new Connection(clusterApiUrl("devnet"));
+
+const PORT = 8080;
+const BASE_URL = 'http://localhost:8080';
+
+// Express app setup
+const app = express();
+app.use(express.json());
+app.use(cors({
+  origin: '*', 
+  methods: ['GET', 'POST', 'OPTIONS'],
+  allowedHeaders: ['Content-Type', 'Authorization', 'Content-Encoding', 'Accept-Encoding'],
+}));
+
+// Routes
+app.get('/actions.json', getActionsJson);
+app.get('/api/actions/transfer-sol', getTransferSol);
+app.post('/api/actions/transfer-sol', postTransferSol);
+
+// Route handlers
+function getActionsJson(req, res) {
+  const payload = {
+    rules: [
+      { pathPattern: "/*", apiPath: "/api/actions/*" },
+      { pathPattern: "/api/actions/**", apiPath: "/api/actions/**" },
+    ],
+  };
+  res.json(payload);
+}
+
+async function getTransferSol(req, res) {
+  try {
+    const { toPubkey } = validatedQueryParams(req.query);
+    const baseHref = `${BASE_URL}/api/actions/transfer-sol?to=${toPubkey.toBase58()}`;
+
+    const payload = {
+      title: "Actions Example - Transfer Native SOL",
+      icon: "https://solana-actions.vercel.app/solana_devs.jpg",
+      description: "Transfer SOL to another Solana wallet",
+      links: {
+        actions: [
+          { label: "Send 1 SOL", href: `${baseHref}&amount=1` },
+          { label: "Send 5 SOL", href: `${baseHref}&amount=5` },
+          { label: "Send 10 SOL", href: `${baseHref}&amount=10` },
+          {
+            label: "Send SOL",
+            href: `${baseHref}&amount={amount}`,
+            parameters: [
+              { name: "amount", label: "Enter the amount of SOL to send", required: true },
+            ],
+          },
+        ],
+      },
+    };
+
+    res.json(payload);
+  } catch (err) {
+    handleError(res, err);
+  }
+}
+
+async function postTransferSol(req, res) {
+  try {
+    const { amount, toPubkey } = validatedQueryParams(req.query);
+    const { account } = req.body;
+
+    if (!account) {
+      throw new Error('Invalid "account" provided');
+    }
+
+    const fromPubkey = new PublicKey(account);
+    const minimumBalance = await connection.getMinimumBalanceForRentExemption(0);
+    
+    if (amount * LAMPORTS_PER_SOL < minimumBalance) {
+      throw new Error(`Account may not be rent exempt: ${toPubkey.toBase58()}`);
+    }
+
+    const transaction = new Transaction().add(
+      SystemProgram.transfer({
+        fromPubkey,
+        toPubkey,
+        lamports: amount * LAMPORTS_PER_SOL,
+      })
+    );
+    transaction.feePayer = fromPubkey;
+
+    const { blockhash } = await connection.getLatestBlockhash();
+    transaction.recentBlockhash = blockhash;
+
+    const serializedTransaction = transaction.serialize({ requireAllSignatures: false });
```

Comment:
Can you update this to use the `createPostResponse` helper function:
https://solana-developers.github.io/solana-actions/functions/createPostResponse.html

Reply:
I'm getting the following error when I try to start the express server after installing and importing `@solana/actions`.

```
> express@1.0.0 start
> node server

node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/johnliu/code/solana-actions-fork/examples/express/node_modules/@solana/actions/lib/esm/constants' imported from /Users/johnliu/code/solana-actions-fork/examples/express/node_modules/@solana/actions/lib/esm/createPostResponse.js
    at new NodeError (node:internal/errors:405:5)
    at finalizeResolution (node:internal/modules/esm/resolve:327:11)
    at moduleResolve (node:internal/modules/esm/resolve:946:10)
    at defaultResolve (node:internal/modules/esm/resolve:1132:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:835:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v18.18.0
```

Installed with:
```
npm i @solana/actions  
```

Package.json dependencies look like this:
```
  "dependencies": {
    "@solana/actions": "^1.0.3",
    "@solana/web3.js": "^1.94",
    "cors": "^2.8.5",
    "express": "^4.19.2"
  }
 ```
 
 Importing like this in server.js:
 ```
 import {
  createPostResponse,
} from "@solana/actions";
```

Any idea how to fix?

Reply:
works with `"@solana/actions": "^1.1.1"`